### ISSUE TITLE
Enhancement: Gaussian Blur Text Background

### ISSUE THREAD

**Issue by ThePragmaticArt** [MattsShack/Plex-Movie-Poster-Display#88]
Currently, the font color of text is configured as a static value making the outline the only way to distinguish the text from the background poster colors from a contrast ratio perspective (note white on white or black on black can be extremely unfriendly depending on the font used). Gaussian Blurs and other background blending techniques can be used to guarantee a darker/lighter background for the text to rest on always ensuring some level of contrast difference for a nice pop effect. Apple has been using this particular technique for quite some time now. Would it be possible to add the option for some guaranteed background contrast enhancer for text?

**Comment by ZeroCoolGOS** [comment]
This is an interesting idea. 

Do you have any sample screenshots or links?

**Comment by thecodingarts** [comment]
Absolutely, here are some examples I was able to pull fairly quickly (note the same effect can be found all over macOS's menus like the quick app switcher).

The first screenshot shows a gaussian blur being used, the second is a gradient blend with a gaussian blur as well. This post may help with some other techniques/options: https://blog.iamsuleiman.com/techniques-to-display-text-overlay-background-images/.

<img width="2032" alt="Screen Shot 2021-05-29 at 7 40 54 PM" src="https://user-images.githubusercontent.com/4037188/120087440-f19ace00-c0b5-11eb-83be-f3e0113ca470.png">
<img width="2032" alt="Screen Shot 2021-05-29 at 7 36 27 PM" src="https://user-images.githubusercontent.com/4037188/120087449-0a0ae880-c0b6-11eb-9fc7-ab563bc8943e.png">
